packages/backend/src/tools/updateDiffHistory.ts

```diff
@@ -316,3 +324,14 @@ function findDescription(
 
   return lines.slice(index, lastIndex).join('\n')
 }
+
+async function updateHashes(projectName: string, chainName: string) {
+  const databasePath = 'discovery/discoveredHashes.json'
+  const chainId = ChainId.fromName(chainName)
+  const shaSum = await getDiscoveryHash(projectName, chainId)
+  const database = getHashesDatabase(databasePath)
+
+  database[getHashesDatabaseKey(projectName, chainId)] = shaSum
+
+  writeFileSync(databasePath, JSON.stringify(database, null, 2))
```

Comment:
This is an unexpected place to write hashes to the file, since reading is implemented in `hashDatabase.ts`. I think that functionality of this whole function should be encapsulated in `hashDatabase.ts`, so that here you simple call one function e.g. `updateHash(projectName, hashName)`.